data/human/human missions.txt

```diff
@@ -5547,3 +5547,221 @@ mission "Quicksilver Mixup 1"
+			`	Deciding not to ask Douglas anything, you look out the windows of the taxi. You're in the midst of New Sydney, and skyscrapers stare down at your tiny taxi from each side of the avenue, but despite this it is still quite green and cheery, with trees on each side and people walking to work in the morning.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 1
+				goto taxi
+
+			`	As the taxi drives on a highway on the edge of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers. Far away on the horizon, you can barely notice the smokestacks of factories.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 2
+				goto taxi
+
+			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney; the University of Chiron's campus is on the other side of town. Presumably, to keep the fierce competition between the two institutions from boiling over each other.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 3
+				goto taxi
+
+			label hospital
+			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside, to match the architecture of the surroundings, and strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative and that people should be treated in a place bright and alive instead of cold and white.`
+			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
+			choice
+				`	"We're looking for Anita Pears, where can we find her?"`
+			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
+			`	"That's me," Douglas interjects before you can speak.`
+			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for today; we have a family-only policy, to keep the privacy of our patients."`
```

Comment:
```suggestion
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for the his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
```
`for today` makes me think, "oh, this procedure is day-specific. Maybe this mission should have a deadline?" I think rewording this is a better fix than adding a deadline.

Reply:
```suggestion
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
```